project/src/pages/main/main.tsx

```diff
@@ -23,7 +33,7 @@ function Main({placesCount, offers}: MainProps): JSX.Element {
               <h2 className="visually-hidden">Places</h2>
               <b className="places__found">{placesCount} places to stay in Amsterdam</b>
               <Sort className="places__sorting" />
-              <PlacesList className="cities__places-list" offers={offers} />
+              <PlacesList className="cities__places-list" offers={offers} onMouseOver={onMouseOver}/>
```

Comment:
Лучше более предметно назвать коллбек чтобы было более понятно 
```suggestion
              <PlacesList className="cities__places-list" offers={offers} onCardHover={handleMouseOver}/>
```